CHANGELOG.md

```diff
@@ -46,6 +46,8 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
   ([PR](https://github.com/hashicorp/boundary-ui/pull/2237))
 * ui: Add UI support for MinIO (Enterprise and HCP Boundary only)
   ([PR](https://github.com/hashicorp/boundary-ui/pull/2248))
+* Allow descriptions to contain newlines and other whitespace
```

Comment:
This should be moved up to the `Next` section

Reply:
my eyes glazed over and I just added it to the first new and improved section I saw, my bad. fixed